docs/app/pages/components/components-sections/notifications/badge/badge.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component } from '@angular/core';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { IPlayground } from '../../../../../interfaces/IPlayground';
+import { IPlaygroundProvider } from '../../../../../interfaces/IPlaygroundProvider';
+
+@Component({
+    selector: 'uxd-components-badge',
+    templateUrl: './badge.component.html',
+    styleUrls: ['./badge.component.less'],
+})
+@DocumentationSectionComponent('ComponentsBadgeComponent')
+export class ComponentsBadgeComponent extends BaseDocumentationSection implements IPlaygroundProvider {
+    hideButton: boolean = true;
+
+    playground: IPlayground = {
+        files: {
+            'app.component.html': this.snippets.raw.appHtml,
+            'app.component.ts': this.snippets.raw.appTs,
+        },
+        modules: [
+            {
+                imports: ['BadgeModule'],
+                library: '@ux-aspects/ux-aspects',
+            },
+            {
```

Comment:
I don't think router should be required here?

Reply:
Hey, yeah it seems it is. The whole page breaks without it.

Reply:
What's the error? Why is it required?